wg-meetings/2022-07-26.md

```diff
@@ -0,0 +1,52 @@
+# Node.js  Diagnostics WorkGroup Meeting 2022-07-26
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=_GILHdoV_Fc
+* **GitHub Issue**: https://github.com/nodejs/diagnostics/issues/574
+* **Minutes Google Doc**: https://docs.google.com/document/d/1l_-UyilTMA3O9YGQMvPfA5Iuz7_5SZc0WmdSfk2zFzM/edit
+
+## Present
+
+* Diagnostics team: @nodejs/diagnostics
+* Stephen Belanger: @qard
+* Rafael Gonzaga: @rafaelgss
+* Anton Whalley: @No9
+
+## Agenda
+
+## Announcements
+
+*Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/diagnostics
+
+* alternatives to `llnode` [#571](https://github.com/nodejs/diagnostics/issues/571)
+  * Anton will open a new issue with the next steps to fully support `llnode`
+  * This issue will be on the `diag-agenda` to keep tracking the work
+
+* Re-evaluate eBPF Tracing tool tier-list [#535](https://github.com/nodejs/diagnostics/issues/535)
+  * Rafael is still investigating it
+  * Anton and Mary did research on this topic in the past.
```

Comment:
Discussion from the 2020-05-06 Diagnostics Meeting here https://github.com/nodejs/diagnostics/pull/388